content/how.njk

```diff
@@ -0,0 +1,125 @@
+  </li>
+</ul>
+
+<h4 id="delivery">Delivery</h4>
+
+<p>
+  In this phase we deliver the work agreed in the Discovery phase to the standards defined in <a href="#dod">Definition of done</a>.
+</p>
+
+<p>
+  While we aim to break down work, where Discovery should have uncovered unknowns, our agile practices will allow us to change course in the face of changing requirements or priorities or uncovering "unknown unknowns".
+</p>
+
+<h4 id="reflection">Reflection and planning</h4>
+
+<p>
+  Due to our agile practices, demonstrations and gathering feedback happens continuously, however this final phase allows for a deeper assessment of the work delivered.
+</p>
+
+<p>
+  It is also when any future work is planned and agreed upon, that can kick off a new Discovery phase.
+</p>
+
+<h3 id="decision-making">Decision making</h3>
+
+<p>
+  Decisions are made daily, about architecture, implementation, future direction, technology choices, etc.
+  We believe that capturing these in a lightweight, but somewhat formalised format has benefits, not just in enabling better decisions, but also as a reference for the future and new team members.
+  Hence, we use Decision Logs, which is an iteration on <a href="https://agilemanifesto.org/principles.html">Requests for comment (RFCs)</a> and <a href="https://docs.aws.amazon.com/prescriptive-guidance/latest/architectural-decision-records/adr-process.html">Architectural decision records (ADRs)</a>.
+  A typical decision log will capture:
+</p>
+
+<ul>
+  <li>
+    <strong>Context:</strong>
+    This captures what the decision is about and why it is important.
+    It also defines the criteria for the decision.
+  </li>
+  <li>
+    <strong>Options:</strong>
+    Decisions are usually an appraisal of different options.
+    These could range from various technology choices and approaches to binary choices like whether something needs to be done or not.
+    This section captures these options with some assessment based on the criteria defined in "Context".
+  </li>
+  <li>
+    <strong>Decision:</strong>
+    This section will capture the option that was chosen.
+    While the first two sections may be written by a single individual this will usually be done following a discussion with the wider team or stakeholders.
+  </li>
+</ul>
+
+<h3 id="dod">Definition of done</h3>
+
+<p>
+  We consider work "done" when it:
+</p>
+
+<ul>
+  <li>Meets the user or acceptance criteria defined.</li>
+  <li>Passes quality assurance.</li>
+  <li>Is documented as applicable.</li>
+  <li>Goes through all environments and is released to users.</li>
+</ul>
+
+<h3 id="standards">Technical standards</h3>
+
+<p>
+  We use standards, such as <a href="https://www.conventionalcommits.org/en/v1.0.0/">conventional commits</a> and <a href="https://conventionalcomments.org/">conventional comments</a> with our code.
+  They enable clearer communication and automation.
```

Comment:
**suggestion:**

Per above:

```suggestion
  We use conventions, such as <a href="https://www.conventionalcommits.org/en/v1.0.0/">conventional commits</a> and <a href="https://conventionalcomments.org/">conventional comments</a> with our code.
  They enable clearer communication and automation.
```

We could also mention:

- [SemVer](https://semver.org/).
- [Keep a CHANGELOG](https://keepachangelog.com/en/1.0.0/) (though, would we always (ever?) do this?).
- [Twelve Factor](https://12factor.net/).
- Something about preferring to use containers for flexible deployment?
- Something about minimising tech footprint?
- ...

Maybe this is more broadly like "engineering principles"?